### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by FragmentedPacket** [netbox-community/ansible_modules#526] [General]
Hi everyone,

I just wanted to provide an update that I'm no longer going to maintain this project and I'm looking for a maintainer/owner to take over. The main reason I'm abandoning ship is due to lack of time and the time I do have I want to spend learning new technologies and skills. Ansible module development is very common at $dayjob and something I no longer have the passion to do this on my own time. 

I really appreciate everyone that I have interacted with over the last few years, the people who contributed to this project, and all the people that have used this collection in production!

I really hope to find a new owner for this collection, but due to it being open source, feel free to fork!

> If you're interested in taking over this project, please reach out to my either on [Twitter](https://twitter.com/FragmentedPkt) or [NTC Slack](https://slack.networktocode.com).

**Comment by dmsimard** [comment]
o/ thanks for your work on the collection, it's appreciated!

I will advertise and talk to folks about it to see if I can find help.

**Comment by dschier-wtd** [comment]
@dmsimard @FragmentedPacket I am able to take over this. I am also discussion this with my current employer, since we are using ansible and netbox in production and for some customers.

**Reply by 2bithacker** [reply]
Hello! Looks like I should have gotten a formal introduction post out sooner. Yesterday I became the new maintainer of the project. I'd certainly appreciate any help with sorting through issues and reviewing PRs though, and welcome any feedback.

**Reply by dschier-wtd** [reply]
Nice, congratulations! Thank you for taking care.

Let's see if/how I can support you.

**Comment by sc68cal** [comment]
This repo is very important to my day to day work. I have been using it quite heavily and will discuss with my employer as well. I will at the very least contribute fixes for things I run across.

**Comment by ryanmerolle** [comment]
Though @2bithacker will likely still contribute, it seems like we need some assistance with getting the backlog under control while providing a maintainer level oversight of various PRs submitted over the long term.  The bar @FragmentedPacket set is quite high, and we want to continue this.

Anyone have any interest in this?  The community organization is looking for 1 or 2 people to help fill this role.

@daniel-wtd or @sc68cal do you happen to have any interest?  The time is likely not as involved as a netbox ("core") maintainer, 

The backlog would not be as overwhelming as it may since since I at least, would be happy to provide some person power to grind through triage, linking issues (duplicate or other wise), and directing people in the discussions.

**Reply by FragmentedPacket** [reply]
I'm more than happy to spend some time with anyone willing to take over the project and answer any questions, etc.

**Reply by rodvand** [reply]
I would be happy to help with this project!

**Reply by sc68cal** [reply]
I use these Ansible modules on a daily basis, so I am very interested in helping, and I will also talk with my employer about reserving part of my work capacity for maintaining these modules.

**Reply by nahun** [reply]
I also use these modules on a daily basis and would like to help if I can. I've contributed to the inventory module a bunch already.

I'll talk to my employer about work capacity as well, but I wouldn't hold your breath...

**Comment by ghost** [comment]
So, is there any need for more maintainers?

**Reply by ryanmerolle** [reply]
Yes, we are still looking for maintainers.  We have not made any official announcements yet, but there is some momentum with low hanging fruit contributions, fixing up the CI, and further testing with NetBox 3.0.

There are a few [issues I labeled as next release milestone](https://github.com/netbox-community/ansible_modules/milestone/6) to track the pressing items, we would love assistance there, and its frankly a good way to demonstrate ones ability to contribute in a more meaningful role such as a maintainer role.  We do not want to just jump at handing out the role if people cannot commit to it or roll up their sleeves to at least help the project through the current backlog.
